source/webp-machine.ts

```diff
@@ -68,11 +67,12 @@ export class WebpMachine {
 				return
 			}
 			try {
-				const webpData = await loadBinaryData(src)
+				const webpData = isBase64Url(src)
+					? convertDataURIToBinary(src)
+					: await loadBinaryData(src)
```

Comment:
```if (this.detectWebpImage(image))```: this line checks if the ```src``` contains ".webp". Does a base64 webp src contain ".webp"? I haven't tested the code sorry but just by looking at it, it seems this code won't be reached. Maybe you can modify the regex there?

Reply:
thanks for this reply. yes, this is a current deficiency of webp-hero, the webp images are detected by their filename's extension and must be `.webp`... it would be better if we could find better means to detect this, perhaps mime type or something like that